# Calculate format=diff os_install_arch_machine==i686&&merge(www-client/firefox)=>68,68.1
diff --git a/firefox-68.0/gfx/skia/skia/third_party/skcms/src/Transform_inl.h b/firefox-68.0/gfx/skia/skia/third_party/skcms/src/Transform_inl.h
index 69efc98..25f1e63 100644
--- a/firefox-68.0/gfx/skia/skia/third_party/skcms/src/Transform_inl.h
+++ b/firefox-68.0/gfx/skia/skia/third_party/skcms/src/Transform_inl.h
@@ -559,7 +559,7 @@ SI void sample_clut_16(const skcms_A2B* a2b, I32 ix, F* r, F* g, F* b) {
 
 // GCC 7.2.0 hits an internal compiler error with -finline-functions (or -O3)
 // when targeting MIPS 64,  I think attempting to inline clut() into exec_ops().
-#if 1 && defined(__GNUC__) && !defined(__clang__) && defined(__mips64)
+#if 1 && defined(__GNUC__) && !defined(__clang__) && (defined(__mips64) || defined(__i386))
     #define MAYBE_NOINLINE __attribute__((noinline))
 #else
     #define MAYBE_NOINLINE
